### ISSUE TITLE
Documentation on getyank: examples missing?

### ISSUE THREAD

**Issue by foxth** [choderalab/yank#1177]
Just installed yank with "conda install yank".

Now I wonder where I can find the examples that are shown in the getyank.org/latest/quickstart.html page. I can locate the p-xylene-implicit data files somewhere deep down in my anaconda tree, but not the corresponding yaml file.

So either the docu or the directory structure needs an update...

**Comment by jchodera** [comment]
We've moved the YANK examples to a separate repo, but seem to have neglected to update the quickstart! You can find them here:
https://github.com/choderalab/yank-examples

We do need to clean up this repo a bit before it's really useful---that's on our list for getting the first YANK paper ready!

**Comment by andrrizzi** [comment]
Yes, and I think those examples require a couple of fixes after the last changes to the YAML format. We need to activate automatic testing of those examples.

**Comment by foxth** [comment]
Any idea on the timelines until I can try out things without too much hassle?

**Comment by andrrizzi** [comment]
I don't think we have a specific timeline, but I will progressively try to get to all these issues in the next few months, while we're working on the YANK publication.
